ast/misc.go

```diff
@@ -626,6 +627,26 @@ type HandleRange struct {
 	End   int64
 }
 
+// ShowSlowType defines the type for SlowLog.
+type ShowSlowType int
+
+const (
+	// ShowSlowTop is a ShowSlowType constant.
+	ShowSlowTop ShowSlowType = iota
+	// ShowSlowRecent is a ShowSlowType constant.
+	ShowSlowRecent
+)
+
+// ShowSlow is used for the following command:
+//	admin show slow top [ internal | all] N
+//	admin show slow recent N
+type ShowSlow struct {
+	Tp    ShowSlowType
+	Count uint64
+	// May be "internal" or "all", default is ""
```

Comment:
what about only showing general type?

Reply:
What do you mean by "general" type? is that "user + internal"?
"user" == "" (the default)
"user + internal" == "all"

Reply:
Please add some comment to explain what  `"user"` is, and the default.

Reply:
I've defined a `ShowSlowKind` type, instead of using string. @winkyao @crazycs520